src/app/auth/views/Auth/SignupBlog.tsx

```diff
@@ -3,43 +3,69 @@ import InternxtDevices from '../../../../assets/images/banner/Internxt-secure-cl
 import { Helmet } from 'react-helmet-async';
 
 const textContent = {
-  email: 'Email',
-  passwordLabel: 'Password',
-  emailEmpty: 'Email cannot be empty',
-  passwordLabelEmpty: 'Password cannot be empty',
-  buttonText: 'Get up to 10GB - For free!',
-  legal: {
-    line1: 'By creating an account you accept the',
-    line2: 'terms of service, and privacy policy',
+  en: {
+    title: {
+      line1: 'Make privacy a priority and join',
+      blueText: ' Internxt ',
+      line2: 'today',
```

Comment:
If the text has been added to the translation files, what is this needed for?

Reply:
My bad.